pyaedt/generic/general_methods.py

```diff
@@ -289,9 +289,9 @@ def check_numeric_equivalence(a, b, relative_tolerance=1e-7):
     return True if reldiff < relative_tolerance else False
 
 
-@pyaedt_function_handler()
-def _check_path(path_to_check):
-    return path_to_check.replace("\\", "/") if path_to_check[0] != "\\" else path_to_check
+@pyaedt_function_handler(path_to_check="path")
```

Comment:
I don't think we need to decorate with kwargs internal methods

Reply:
Agreed, do we have a way to know if some users use any of them ?

Reply:
if they use it is at their risk. The intent of a private method should be to not be used by the end user

Reply:
I think this discussion is related to https://github.com/ansys/pyaedt/issues/4666